code/modules/mob/living/carbon/human/species_types/IPC.dm

```diff
@@ -0,0 +1,177 @@
+/datum/species/ipc // Beestation's IPCs
+	name = "IPC" //inherited from the real species, for health scanners and things
+	id = "ipc"
+	say_mod = "states" //inherited from a user's real species
+	sexes = 0
+	species_traits = list(NOTRANSSTING,NOEYESPRITES,NO_DNA_COPY,NOBLOOD,ROBOTIC_LIMBS,NOZOMBIE,MUTCOLORS,REVIVESBYHEALING,NOHUSK,NOMOUTH) //all of these + whatever we inherit from the real species
+	inherent_traits = list(TRAIT_RESISTCOLD,TRAIT_VIRUSIMMUNE,TRAIT_NOBREATH,TRAIT_RADIMMUNE,TRAIT_LIMBATTACHMENT)
+	inherent_biotypes = list(MOB_ROBOTIC, MOB_HUMANOID)
+	mutant_brain = /obj/item/organ/brain/positron
+	mutanteyes = /obj/item/organ/eyes/robotic
+	mutanttongue = /obj/item/organ/tongue/robot
+	mutantliver = /obj/item/organ/liver/cybernetic/upgraded/ipc
+	mutantstomach = /obj/item/organ/stomach/cell
+	mutantears = /obj/item/organ/ears/robot
+	mutant_organs = list(/obj/item/organ/cyberimp/arm/power_cord)
+	mutant_bodyparts = list("ipc_screen", "ipc_antenna", "ipc_chassis")
+	default_features = list("mcolor" = "#7D7D7D", "ipc_screen" = "Static", "ipc_antenna" = "None", "ipc_chassis" = "Morpheus Cyberkinetics(Greyscale)")
+	meat = /obj/item/stack/sheet/plasteel{amount = 5}
+	skinned_type = /obj/item/stack/sheet/iron{amount = 10}
+	exotic_blood = "oil"
+	damage_overlay_type = "synth"
+	limbs_id = "synth"
+	mutant_bodyparts = list("ipc_screen", "ipc_antenna", "ipc_chassis")
+	default_features = list("ipc_screen" = "BSOD", "ipc_antenna" = "None")
+	burnmod = 2
+	heatmod = 1.5
+	brutemod = 1
+	toxmod = 0
+	clonemod = 0
+	staminamod = 0.8
+	siemens_coeff = 1.5
+	reagent_tag = PROCESS_SYNTHETIC
+	species_gibs = "robotic"
+	attack_sound = 'sound/items/trayhit1.ogg'
+	allow_numbers_in_name = TRUE
+	deathsound = "sound/voice/borg_deathsound.ogg"
+	var/saved_screen //for saving the screen when they die
+	var/list/initial_species_traits //for getting these values back for assume_disguise()
+	var/list/initial_inherent_traits
+	changesource_flags = MIRROR_BADMIN | WABBAJACK
+	species_language_holder = /datum/language_holder/synthetic
+
+	var/datum/action/innate/change_screen/change_screen
+
+/datum/species/ipc/random_name(unique)
+	var/ipc_name = "[pick(GLOB.posibrain_names)]-[rand(100, 999)]"
+	return ipc_name
+
+/datum/species/ipc/on_species_gain(mob/living/carbon/C) // Let's make that IPC actually robotic.
+	. = ..()
+	var/obj/item/organ/appendix/appendix = C.getorganslot("appendix") // Easiest way to remove it.
+	appendix.Remove(C)
+	QDEL_NULL(appendix)
+	ADD_TRAIT(C, TRAIT_XENO_IMMUNE, "xeno immune") //makes the IPC immune to huggers
```

Comment:
Isn't the thing is quotes supposed to sort of describe the source of the trait, not the trait itself? I could be wrong, though.